### DISCUSSION TITLE
Combine multiple pre-rendered HTML

### DISCUSSION THREAD

**Discussion by totto2727** [master-co/css#366] [Question]
Some applications insert HTML retrieved from the server side into the DOM.
The HTML and Master CSS are always pre-rendered on the server side.

In this case, both the HTML for the initial display and any additional HTML for updating the DOM will be generated with, `id=“master”`.
Therefore, there will be multiple `<style>` with `id=“master”`.

Is there any way to assign an `id` other than `id=“master”` or hook into the Master CSS lifecycle to remove styles pre-rendered at runtime?

## Sample 
- app: https://expensive-snail-59.deno.dev/content
- code: https://github.com/totto2727-org/hono-mastercss/blob/main/demo/main.tsx

**Comment by 1aron** [comment]
@totto2727 Pre-rendering CSS ensures a quick initial display in the browser. Subsequently, any dynamically generated HTML automatically has its classes extracted and CSS generated by the CSS runtime. This is the solution we currently advocate.

Dynamic insertion of multiple `<style>` elements, akin to a Server Streaming effect, is not what Master does. Instead, a single page dynamically updates and shares the same `style#master`.
